locals.tf

```diff
@@ -38,11 +37,6 @@ locals {
     NoExecute        = "NO_EXECUTE"
   }
 
-  # kube config
-  service_account_name        = "${var.prefix}-cluster-admin-sa"
-  cluster_role_binding_name   = "${var.prefix}-cluster-admin-crb"
-  service_account_secret_name = "${var.prefix}-sa-secret"
```

Comment:
Why were these removed?

Reply:
These variables in local.tf were never used so TFLint threw a warning. The effective set of these variables are actually located in `viya4-iac-gcp/modules/kubeconfig/locals.tf` these are unused duplicates.

---

Had to look through the git history, here's some historical context: 
`service_account_name`,`cluster_role_binding_name`,`service_account_secret_name` were added in PR that introduced the Terraform Enterprise changes here:
https://github.com/sassoftware/viya4-iac-gcp/pull/144/files#diff-9a576c4980b4f226b7ef1042670d564777bbb0ed8ae2d1df33afbd03e2991fcdR38

However in that PR these variables were added to both `viya4-iac-gcp/modules/kubeconfig/locals.tf` and `viya4-iac-gcp/locals.tf`. Only the set in `viya4-iac-gcp/modules/kubeconfig/locals.tf` are actually consumed when templating out the kubeconfig.